### ISSUE TITLE
Contour connection issues

### ISSUE THREAD

**Issue by tamcy** [adobe-fonts/source-han-sans#515]
Can't believe I missed such frequently used character the last time when I checked the contour issues of glyphs with a 辶 in it.

Happens in uni9059-HK, the EL master:
![image](https://github.com/user-attachments/assets/0f2ec5cd-f904-4275-b0c6-bc79ba062f0e)

**Comment by punchcutter** [comment]
There are hundreds of issues like this and they don't really need new issues. We could have a single issue for these. I've fixed a ton so far, but I'm not sure it's really worth documenting every single one.

**Comment by tamcy** [comment]
### 𤩦 u24A66-HK (ExtraLight)

<img width="400" alt="u24A66-HK" src="https://github.com/user-attachments/assets/385aaa90-f1ce-42c0-b7ca-5d521fcdb267" />

---

### 㗀 uni35C0-CN (ExtraLight)
<img width="400" alt="uni35C0-CN" src="https://github.com/user-attachments/assets/8c5f2b5c-cf0c-4e2d-bb71-0c55d9879bc8" />

---

### 樫 uni6A2B-JP (ExtraLight)
<img width="400" alt="uni6A2B-JP" src="https://github.com/user-attachments/assets/8c86b920-2d10-4379-b033-fd2fb38b2e65" />

---

### 潿 uni6F7F-CN (ExtraLight)
<img width="400" alt="uni6F7F-CN" src="https://github.com/user-attachments/assets/63cd2aaa-8a2c-4273-a68f-a3227d996390" />

---

### 葍 uni844D-JP and uni844D-TW (ExtraLight)
<img width="400" alt="uni844D-JP" src="https://github.com/user-attachments/assets/a434426e-041d-413c-bbb4-a7dc209ec8c9" />

---

### 轋 uni8F4B-TW (ExtraLight)

<img width="400" alt="uni8F4B-TW" src="https://github.com/user-attachments/assets/8e82a62b-0c92-4052-92db-aabc462ee6b0" />

**Comment by tamcy** [comment]
Spot a rather weird connection issue in the ExtraLight master of the following glyphs:

```
㫆 uni3AC6-CN
㫉 uni3AC9-CN 
㫊 uni3ACA-CN
㫋 uni3ACB-CN
㫍 uni3ACD-CN
斻 uni65BB-CN
旀 uni65C0-CN
旈 uni65C8-CN
旌 uni65CC-CN
旐 uni65D0-CN
旑 uni65D1-CN
旓 uni65D3-CN
旛 uni65D9-CN
```

Apparantly the 𠃌 components in the above glyphs are not connecting well:
![image](https://github.com/user-attachments/assets/25cfe759-66e9-403e-8633-f5ab62a054bc)

But I suspect the cause is that the 方 component was inadvertently rotated counterclockwise by the designer (and copy and pasted to other glyphs), because the horizontal strokes are also slightly slanted, offseted by 1 pixel:
![image](https://github.com/user-attachments/assets/73a493f8-6b67-4b89-bce7-0b8f0e28cf3e)

**Comment by tamcy** [comment]
Not really a contour connection issue, but a direction one.

- uni71DB-CN 燛
- uni81E6-CN 臦

In this two glyphs, the contour direction of the *flipped-臣* component is incorrect. FWIW, there are two other glyphs, uni71DB-TW and uni81E9-JP, contains a *flipped-臣* component but with the correct direction.

**Comment by punchcutter** [comment]
Yes, those got rotated a tiny bit for no good reason. They aren't the only components like that. I already fixed a bunch of others with bad rotation or skew values.

**Comment by tamcy** [comment]
In v2.005, items in my first 3 reports are missed. Here's a summary on the connection issues:


遙 uni9059-HK (ExtraLight) : top right in 𠂊
𤩦 u24A66-HK (ExtraLight) : first 𡿨 of 幺 
㗀 uni35C0-CN (ExtraLight) : bottom right
樫 uni6A2B-JP (ExtraLight): bottom vertical stroke in 臣 
潿 uni6F7F-CN (ExtraLight): bottom right of the big 囗
葍 uni844D-JP and uni844D-TW (ExtraLight): right of the middle horizontal stroke in 田
轋 uni8F4B-TW (ExtraLight): middle horizontal stroke in 辶
鵮 uni9D6E-CN (ExtraLight): right of middle horizontal stroke  of 日 inside 鳥
㘘 uni3618-CN (ExtraLight): bottom left of 口
遜 uni905C-HK (ExtraLight):first 𡿨 of 幺 

Other items are fixed.
